cli/src/commands/config/list.rs

```diff
@@ -73,9 +75,13 @@ pub fn cmd_config_list(
 ) -> Result<(), CommandError> {
     let template = {
         let language = config_template_language(command.settings());
+        let default_template = match args.show_origin {
+            true => "templates.config_list_detailed",
+            false => "templates.config_list",
+        };
```

Comment:
Naming aside, can't we use `-Tbuiltin_config_list_detailed`? That would require a bit of typing, but we can avoid inconsistency.